package.json

```diff
@@ -50,6 +50,7 @@
     "sinon": "^1.17.3",
     "travis-after-all": "^1.4.4",
     "validate-commit-msg": "2.6.1",
+    "which": "^1.2.8",
```

Comment:
It's being used at https://github.com/sarbbottam/eslint-find-rules/pull/72/files/922dd7e8ce917e9bf24b1662436c6c3f333063d0#diff-7f94494686a3b958a7c65884d555437fR5